### DISCUSSION TITLE
[BUG?] The rendering of the headings divider is strange.

### DISCUSSION THREAD

**Discussion by Ai-Jani** [Akifyss/obsidian-border#264] [Q&A]
(I'm not good at English. sorry)
Hello, thank you for this wonderful theme!
I encountered a small bug and would like to share that information.

When the following two items are set by Style Settings, the headings divider will be displayed incorrectly in only Reading View.
1. Editor - Headings - Remove headings indicator: ON
2. Editor - Headings - Level n Headings - Enable Hn divider: ON

Most of the headings divider are not visible. And the visible lines are in a strange position.
The image below shows the same file opened in Editing View and Reading View.
![Drawing 2024-05-07 22 03 20 excalidraw](https://github.com/Akifyss/obsidian-border/assets/150241164/b83a7e6c-d79c-4615-8b6e-2422c34726a8)
This issue has been confirmed to occur on my Windows PC and Android.

Please check it, if you sure.
Thank you.

**Comment by Akifyss** [comment]
The new version has fixed this issue. You can try updating it.

**Reply by Ai-Jani** [reply]
It worked properly.
thank you so much!
